02_knight_and_princess/novikov_dmitry/graph_generation.hpp

```diff
@@ -61,7 +61,8 @@ void generate_blue_edges(Graph& graph) {
   // учитывать
   for (Depth current_depth = 1; current_depth <= graph.get_depth();
        ++current_depth) {
-    const auto& vertices_at_depth = graph.get_vertices_at_depth(current_depth);
+    const auto& vertices_at_depth =
+        static_cast<const Graph&>(graph).get_vertices_at_depth(current_depth);

```

Comment:
Зачем везде добавил `static_cast`?
```suggestion
    const auto& vertices_at_depth = graph.get_vertices_at_depth(current_depth);
```